### PR TITLE
feat: get customer address

### PR DESCRIPTION
**Pull request by justyna-13**
### 🔗 Linked issue
https://vsf.atlassian.net/browse/SV-146

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

composable for getting customer address (temporary mocked) 

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/web/composables/useCustomerAddress/types.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import type { Ref } from 'vue';
 import type { Maybe, SfAddress } from '@vue-storefront/unified-data-model';
 
-export interface UseAddressState {
+export interface UseCustomerAddressState {
   data: Maybe<SfAddress>;
   loading: boolean;
 }
 
 export type FetchAddress = () => Promise<Ref<Maybe<SfAddress>>>;
```
Comment (Szymon-dziewonski) [inline apps/web/composables/useCustomerAddress/types.ts:9]:
```suggestion
export type FetchCustomerAddress = () => Promise<Ref<Maybe<SfAddress>>>;
```

File: apps/web/composables/useCustomerAddress/types.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
-export interface UseAddress {
-  data: Readonly<Ref<UseAddressState['data']>>;
+export interface UseCustomerAddress {
+  data: Readonly<Ref<UseCustomerAddressState['data']>>;
   loading: Readonly<Ref<boolean>>;
   fetchAddress: FetchAddress;
```
[the remark this answers is not in the document]
Comment (justyna-13) [reply apps/web/composables/useCustomerAddress/types.ts]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/104/commits/e7fee15b9ed0c10a67c681585afbe3148c840c18

File: apps/web/composables/useCustomerAddress/types.ts (hunk 1 above)
Comment (justyna-13) [reply apps/web/composables/useCustomerAddress/types.ts]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/104/commits/e7fee15b9ed0c10a67c681585afbe3148c840c18

File: apps/web/composables/useCustomerAddress/useCustomerAddress.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ export const useCustomerAddress: UseCustomerAddressReturn = () => {
    * @example
    * fetchAddress();
```
Comment (Szymon-dziewonski) [inline apps/web/composables/useCustomerAddress/useCustomerAddress.ts:22]:
```suggestion
   * fetchCustomerAddress();
```

Comment (Szymon-dziewonski) [reply apps/web/composables/useCustomerAddress/useCustomerAddress.ts:22]: and in line 12 same 
```
const { data, loading, fetchCustomerAddress } = useCustomerAddress();
```

**Review by Szymon-dziewonski — APPROVED** [review]

**Pull request merged**